internal/internal_workflow_task_handler_mock.go

```diff
@@ -41,7 +41,15 @@ func (_m *MockWorkflowTaskHandler) EXPECT() *MockWorkflowTaskHandler_Expecter {
 func (_m *MockWorkflowTaskHandler) ProcessWorkflowTask(task *workflowTask, f decisionHeartbeatFunc) (interface{}, error) {
 	ret := _m.Called(task, f)
 
+	if len(ret) == 0 {
+		panic("no return value specified for ProcessWorkflowTask")
+	}
+
 	var r0 interface{}
+	var r1 error
+	if rf, ok := ret.Get(0).(func(*workflowTask, decisionHeartbeatFunc) (interface{}, error)); ok {
+		return rf(task, f)
+	}
```

Comment:
Below there is this:
```go
if ret.Get(0) != nil {
	r0 = ret.Get(0).(interface{})
}
```
which should have panicked if the new signature was used, so I believe this is a safe addition / nobody could have been passing this value in previously.